src/wp-includes/media.php

```diff
@@ -1903,6 +1910,44 @@ function wp_filter_content_tags( $content, $context = null ) {
 	return $content;
 }
 
+/**
+ * Adds a fallback mechanism to replace webp images with jpeg alternatives on older browsers.
+ *
+ * @since 6.1.0
+ */
+function wp_wepb_fallback() {
+
+	ob_start();
+	$suffix     = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+	$script_src = includes_url( "js/webp-fallback$suffix.js" );
+
+	?>
+	( function( d, i, s, p ) {
+		s = d.createElement( s );
+		s.src = '<?php echo esc_url_raw( $script_src ); ?>';
+
+		i = d.createElement( i );
+		i.src = p + 'jIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==';
+		i.onload = function() {
+			i.onload = undefined;
+			i.src = p + 'h4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
+		};
+
+		i.onerror = function() {
+			d.body.appendChild( s );
+		};
+	} )( document, 'img', 'script', 'data:image/webp;base64,UklGR' );
+	<?php
+	$javascript = ob_get_clean();
+	wp_print_inline_script_tag(
+		preg_replace( '/\s+/', '', $javascript ),
```

Comment:
See my above comment, here it would be cleaner to use `file_get_contents()` from an actual JS file, and before that inject the dynamic piece of data into a global.